src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Gauges.cpp

```diff
@@ -0,0 +1,8 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Gauges.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+GaugeCondition::GaugeCondition(CkMigrateMessage* msg) : PUP::able(msg) {}
+}  // namespace GeneralizedHarmonic::gauges
```

Comment:
Just so I understand, I wonder why you need this file at all?

Reply:
I want to define the constructor in a `cpp` file instead of an `hpp`